book/website/requirements.txt

```diff
@@ -5,5 +5,7 @@ numpy==1.23.2
 matplotlib==3.5.2
 datascience
 folium
+sphinx-panels
 jupyter-book==0.13
-jinja2>3.1
\ No newline at end of file
+jinja2>3.1
+git+https://github.com/alan-turing-institute/bio-Turing-Way.git@main#subdirectory=pathways
```

Comment:
Now that we have an org. It would be good to take that package and move it to a new repository in our org.

Perhaps it is a project the infrastructure WG should adopt and maintain?

Reply:
> Perhaps it is a project the infrastructure WG should adopt and maintain?

I think a lot of my concern in my above comment is that it will be assumed anyway that the infra WG are maintaining it - so we need to be onboarded to it, but we are also stretched thin.

Reply:
Thanks both. I appreciate the feedback. My read of the situation is if we can find a maintainer outside the infra WG, that would make the decision to merge this feature easier. Is that right?

Reply:
@malvikasharan my thoughts are, The Turing Way should maintain the code and the infra WG seem like the best people to do that.

I think _we_ should invest in it because it adds value to the book and we will depend on it. Longer term, it would be really great if we build on the work to make a more general pathways plugin for Jupyterbook/Sphinx.

Like @sgibson91 said having some onboarding is probably the most important thing, so that we are best prepared to maintain it. I feel fairly comfortable with the infra WG taking responsibility for it. I think it's certainly a better situation than hoping or expecting someone outside of the project will continue to support it.

Reply:
Updated this in ab89137 to point to https://github.com/the-turing-way/pathways v0.1.0.

Looks fine to me in the preview.